README.md

```diff
@@ -24,10 +24,13 @@ Contains crates specific to our application. Can depend on libraries located in
 ## Setup build environment
 
 1. Install `rustup`: `curl https://sh.rustup.rs -sSf | sh`
-2. Install libzmq:
+2. Install the following versions of Rust
+    - stable: `rustup install stable`
+    - nightly-2019-04-30: `rustup install nightly-2019-04-30` (this one is only used for `rustfmt`)
```

Comment:
📣 move that to the `## Setup testing/dev environment` section

Reply:
Good idea, will do that!